tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/variables/VariableVisibilityTest.java

```diff
@@ -39,6 +39,8 @@
 /**
  * Test class for variable visibility.
  */
+
+@Test(enabled = false)
```

Comment:
Lets disable only the failing test case ftm.

Reply:
As the tests are failing randomly and identifying and disabling every failing test case is time-consuming, we(@NipunaRanasinghe and I) decided to disable the complete class until the update-12 PRs are merged.

Reply:
I only noticed intermittent failures in only 2 test cases so far, but its okay to disable the complete variable tests ftm given that this is a release blocker